### ISSUE TITLE
Right to Left for scribefire on chrome

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#238]
```
What's the problem?

I can't see how can I configure scribefire in Google chrome for writing Right 
to Left.



What browser are you using?

Google chrome



What version of ScribeFire are you running?
Last version.
```

-----
Original issue reported on code.google.com by `moosavyk...@gmail.com` on 26 Jan 2011 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
Not yet supported, but I'll work on it.
```

-----
Original comment by `cfinke` on 2 Feb 2011 at 4:49
* Changed state: **Accepted**
* Added labels: **Type-Enhancement**
* Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Issue 407 has been merged into this issue.
```

-----
Original comment by `cfinke` on 15 Aug 2011 at 2:06

**Comment by GoogleCodeExporter** [comment]
```
Done in r341; will be included in version 1.9.
```

-----
Original comment by `cfinke` on 31 Aug 2011 at 2:52
* Changed state: **Fixed**
* Added labels: **Milestone-1.9**
